### DISCUSSION TITLE
Victory Pose

### DISCUSSION THREAD

**Discussion by UltramanTime** [Techokami/SonicWorldsNext#53] [Q&A] [answered]
Question:
Will there be Victory Pose for All Characters?

**Comment by Techokami** [comment, marked as answer]
They all have victory poses already in their sprite sheets? Have you looked inside the data for this project?

**Reply by UltramanTime** [reply]
Yes

**Comment by DimensionWarped** [comment]
In fairness, it's not implemented in the form of an animation or in an action taken during the end of the stage, it's *just* in the sprite sheet for now.

Still, this is very basic stuff to implement and the exact method of using victory poses is actually unique to each game. I don't think Sonic 1 and 2 even had them, they just either had Sonic run off the screen or else would let you chill with control until the transition. I personally kinda prefer those approaches to having you sit with your character's fingers wagging.
